### ISSUE TITLE
convert any "links" in data to actual URLs

### ISSUE THREAD

**Issue by mapsam** [codeforamerica/rva-searchable-services#7]
Convert strings of URLs into actual hyperlinks using some sort of regex magic

**Comment by mapsam** [comment]
So there's two options here as far as I know:
1. roll our own regex, which @esmithayer is going to have to do because I'm not a wizard. [Something like this](https://github.com/component/regexps/blob/master/index.js#L3)? 
2. find a JS library to do this for us.

I'm interested in testing the latter. Here are some potential options:
- [jQuery Linkify](https://github.com/SoapBox/jQuery-linkify)
- [JS Linkify](https://github.com/cowboy/javascript-linkify)
- [Autolinker.js](https://github.com/gregjacobs/Autolinker.js)

Mind if I play with these, @esmithayer?

**Comment by mapsam** [comment]
Pretty neat stuff, these libraries! I'm currently trying out Autolinker.js and it seems to work quite well. 

Under `init()` I started a new instance of the Autolinker library called `alink` that can be used throughout the scripts. Ran into some issues when passing the newly created string with `<a>` tags using `.innerHTML` since that doesn't covert into DOM nodes and just stays as plain text. Work-around uses a `Handlebar.registerHelper()` function that allows you to pass whatever content through a custom script before it is placed into the Handlebar template. So, new helper exists called `add-links`:

``` javascript
Handlebars.registerHelper('add-links', function(context) {
  var linked = alink.link(context);
  return new Handlebars.SafeString(linked);
});
```

and can be used within the template as such:

```
{{ add-links your_content }}
```

You'll see it being [used only for the "contact" info](https://github.com/codeforamerica/rva-searchable-services/commit/07421f340a4b2648ab7a96109a8bb1ded03b16ee#diff-f5cd5f1dbc981bdb3656ee8adeba8fcbR11) right now, but couldn't hurt to pass descriptions through this filter as well.
